README.md

```diff
@@ -1,5 +1,24 @@
-# Todo MVC with ReactJS + redux-saga + Redux
+# Todo MVC with ReactJS + Redux-saga + Redux
```

Comment:
This is either `redux-saga` or `Redux-Saga`: https://github.com/redux-saga/redux-saga

Reply:
:ok_hand: